cmd/tools/benchmark/job/main.go

```diff
@@ -42,7 +42,7 @@ func main() {
 			runner.WithName(name),
 			runner.WithVersion(info.Version, maxVersion, minVersion),
 			runner.WithConfigLoader(func(path string) (interface{}, *config.GlobalConfig, error) {
-				cfg, err := config.NewConfig(path)
+				cfg, err := config.NewConfig(context.Background(), path)
```

Comment:
It would be better to define ctx variable and should be use same context with runner.Do